### DISCUSSION TITLE
Batch HiRes-Fix from files with parameters

### DISCUSSION THREAD

**Discussion by gobshyte** [vladmandic/sdnext#603] [Ideas]
There is an excellent script here:
https://github.com/thundaga/SD-webui-txt2img-script

In Automatic1111 it allows you to batch process files with HiRes-Fix using the parameters stored inside the files.
I can't see anyway to do this in the new functions in Vlad's, and the script does not work either. :(

I'm afraid fixing the script is beyond me, but wondered if this could be incorporated in some way here?

Also there is another similar script for a subsequent Img2Img processing, here
https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/4938
Again this lets you bath process files using the stored parameters, but within Img2Img, and apply SD Upscale at the same time.

Would be awesome if there was some improved batch processing like this, it makes my workflow a lot more efficient in A1111, but it is so slow compared to Vlad's, I'd love to migrate fully.

**Comment by vladmandic** [comment]
i suggest to contact the script author to create actual extension. custom scripts should be loaded as extension, never manually copied to scripts folder - that will cause quite a few issues with automatic updates as you end up modifying core files that belong to repo.
